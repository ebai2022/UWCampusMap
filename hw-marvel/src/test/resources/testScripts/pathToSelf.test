# Testing the finding path to the same node

LoadGraph g strangers.csv

FindPath g Andrew Andrew

FindPath g Ethan Ethan