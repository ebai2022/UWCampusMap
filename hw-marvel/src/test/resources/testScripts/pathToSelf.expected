# Testing the finding path to the same node

loaded graph g

path from Andrew to Andrew:

path from Ethan to Ethan: